### PR TITLE
fresh readme to reflect styling of app, naming conventions etc

### PR DESCRIPTION
**Pull request by geoknee**
The main thing here is to show of the new look and feel of the app. 
Showing 4 screens could be overkill though?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-## Rock Paper Scissors: A Force-Move Games POC
+## Rock Paper Scissors: an example ForceMove game
+
+This app is an example of a game built on our [ForceMove](https://magmo.com/force-move-games.pdf) protocol. You can play against a friend or against our bots on the ropsten testnet.
+
+![splash](./screens.png 'screens')
+
+To run the app on your machine, clone down the code and follow the instructions below.
```
Comment (kerzhner) [inline README.md]: @geoknee Minor wording tweak. You would usually say "clone" or "pull down," but not "clone down."

File: README.md [hunk 2]
```diff
@@ -16,10 +22,15 @@
...
 
-1. Start ganache
-    * Either using the app
-    * Or by running `ganache-cli` in a different tab
-2. Run the server:
+1. Start ganache by running
+    ```
+    ganache-cli
+    ```
+2. **In a new tab** Run the server
```
Comment (kerzhner) [inline README.md]:
```suggestion
2. **In a new terminal tab** Run the server
```

File: README.md [hunk 3]
```diff
@@ -16,10 +22,15 @@
...
+    ```
+    yarn start
+    ```
+    After seeing the message "compiled successfully!", kill the server <kbd>cmd</kbd>+<kbd>C</kbd> and run it again
```
Comment (kerzhner) [inline README.md]: @geoknee do you know why this step is needed?

Comment (geoknee) [reply README.md]: There is an open issue with a partial explanation.

**Pull request merged**